hazelcast/src/main/java/com/hazelcast/internal/management/operation/ChangeWanStateOperation.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.operation;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.spi.Operation;
+
+import java.io.IOException;
+
+/**
+ * Pauses/Restarts the Wan Replication {@link com.hazelcast.wan.WanReplicationService} on a node. Doesn't block the WAN queue.
+ */
+public class ChangeWanStateOperation extends Operation {
+
+    private boolean start;
+
+    public ChangeWanStateOperation() {
+    }
+
+    public ChangeWanStateOperation(boolean start) {
```

Comment:
I think boolean is not satisfying what we need here. START,STOP like enums could be used. you can ignore it for now but we definitely need an enhancement issue for this and other usages...